packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -87,20 +91,26 @@ describe("Runtime", () => {
                 afterEach(() => {
                     batchBegin = 0;
                     batchEnd = 0;
+                    seqNumber = 0;
                 });
 
+                function processOp(message: Partial<ISequencedDocumentMessage>) {
+                    seqNumber++;
+                    message.sequenceNumber = seqNumber;
+                    deltaManager.inbound.push(message as ISequencedDocumentMessage);
+                }
+
                 it("Single non-batch message", () => {
                     const clientId: string = "test-client";
                     const message: Partial<ISequencedDocumentMessage> = {
                         clientId,
-                        sequenceNumber: 0,
                         type: MessageType.Operation,
                     };
 
                     // Send a non-batch message.
-                    scheduleManager.beforeOpProcessing(message as ISequencedDocumentMessage);
-                    scheduleManager.afterOpProcessing(undefined, message as ISequencedDocumentMessage);
+                    processOp(message);
 
+                    assert.strictEqual(deltaManager.inbound.length, 0, "processed all ops");
```

Comment:
#minor. For consistency, the message should be negative (like 'Did not process all ops')

Reply:
Sure, fixed it in other places as well.